lib/govuk_content_schemas/schema_combiner.rb

```diff
@@ -53,11 +53,18 @@ def add_details(schema)
   end
 
   def add_links(schema, target)
-    base_links = schemas[:base_links]
-    edition_links = schemas[:edition_links]
+    base_linkset_links = schemas[:base_links]
     linkset_links = schemas[:links]
+    base_edition_links = schemas[:base_edition_links]
+    edition_links = schemas[:edition_links]
+
+    order = [
+      edition_links,
+      linkset_links,
+      base_edition_links,
+      base_linkset_links
+    ].compact
 
```

Comment:
ah, right. could require ActiveSupport core extensions and do `slice.values`, but probably not worth it.